data/pokemon/dex_entries/009.asm

```diff
@@ -1,4 +1,4 @@
-	db "Shellfish@" ; species name
+	db "Shell@" ; species name
```

Comment:
Change itself is fine, but I think should only really apply to nonfaithful builds, since Blastoise is still the Shellfish Pokemon as of Gen 9. (Also please make sure you list all changes at the top of the PR! ^^)

Reply:
OK, I made it so it doesn't change if faithful.